model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php

```diff
@@ -101,20 +104,7 @@ private function resolve(ItemMediaResolver $resolver, string $assetUrl, string $
 
     private function getReplacementName(string $link, array $replacementList): string
     {
-        $basename = basename($link);
-
-        $replacement = $basename;
-
-        $count = 0;
-        while (in_array($replacement, $replacementList)) {
-            $dot = strrpos($basename, '.');
-            $replacement = $dot !== false
-                ? substr($basename, 0, $dot) . '_' . $count . substr($basename, $dot)
-                : $basename . $count;
-            $count++;
-        }
-
```

Comment:
Is this the only real change?

Reply:
Nope, as discussed in slack https://github.com/oat-sa/extension-tao-itemqti/pull/1517/files#diff-e8652df0cba6546296d14104113388f6R89 this is required too

Reply:
also : https://github.com/oat-sa/extension-tao-itemqti/pull/1517/files#diff-e8652df0cba6546296d14104113388f6R89